kafka_consumer/datadog_checks/kafka_consumer/datadog_agent.py

```diff
@@ -0,0 +1,17 @@
+# (C) Datadog, Inc. 2020-present
+# All rights reserved
+# Licensed under Simplified BSD License (see LICENSE)
+
+from typing import Optional
+
+try:
+    from datadog_agent import read_persistent_cache, write_persistent_cache
+except ImportError:
+
+    def write_persistent_cache(key, value):
+        # type: (str, str) -> None
+        pass
+
+    def read_persistent_cache(key):
+        # type: (str) -> Optional[str]
+        return ''
```

Comment:
Why this?

Reply:
Sounds good. I copied it from another check but you have more context than me on it.

Reply:
Actually, the base check uses the same trick: https://github.com/DataDog/integrations-core/blob/260ef0d91d7e7cfbe541251833b9d24e80202f3c/datadog_checks_base/datadog_checks/base/checks/base.py#L43-L42

Reply:
If you think it's safe to switch to directly import `datadog_agent`, I'm happy to change it.